cypress/integration/lpa/cases/lpa.spec.js

```diff
@@ -0,0 +1,125 @@
+beforeEach(() => {
```

Comment:
I'm not a fan of having these top-level, especially when you have a nested one for the other describe

Reply:
Yep, fair point. Originally they shared the same setup, but that introduced other problems